apps/examples/wifi_manager/wm_test/wm_test_stress_test5.c

```diff
@@ -0,0 +1,826 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <errno.h>
+#include <sys/socket.h>
+#include <sys/time.h>
+#include <sys/types.h>
+#include <wifi_manager/wifi_manager.h>
+#include <stress_tool/st_perf.h>
+#include <netdb.h>
+#include "wm_test.h"
+#include "wm_test_utils.h"
+#include "wm_test_network.h"
+#include "wm_test_log.h"
+#include "wm_test_stress_test5.h"
+
+#define TAG "[WTS5]"
+#define WM_TEST_TRIAL CONFIG_WIFIMANAGER_TEST_TRIAL
+#define WT_UDP_ADDR CONFIG_LWIP_DHCPS_SERVER_IP
+#define WT_UDP_PORT 8888
+#define WT_TCP_PORT 8889 
+#define WT_BUF_SIZE 1024
+//if semaphore operation failed then it'll try it again 10ms later
+#define WT_SEM_TRY_WAIT_US 10000
+#define HOSTNAME_BUFSIZE 128
+#define BROADCASTING_PORT 8890
+
+static char *WM_AP_SSID;
+static char *WM_AP_PASSWORD;
+static wifi_manager_ap_auth_type_e WM_AP_AUTH;
+static wifi_manager_ap_crypto_type_e WM_AP_CRYPTO;
+static char *WM_SOFTAP_SSID;
+static char *WM_SOFTAP_PASSWORD;
+static int WM_SOFTAP_CHANNEL;
+static uint8_t WM_IS_DTLS;
+static uint32_t WM_DTLS_DATA_SIZE;
+static uint8_t WM_IS_TLS;
+static uint32_t WM_TLS_DATA_SIZE;
+static char *WM_HOSTNAME_FILE;
```

Comment:
If possible how about to initialize the values.

Reply:
Changes Done as requested